godtools/App/Features/AppLanguage/Domain/UseCases/SearchAppLanguageInAppLanguagesListUseCase.swift

```diff
@@ -28,7 +28,6 @@ class SearchAppLanguageInAppLanguagesListUseCase {
                 if searchText.isEmpty {
                     
                     return Just(languageItems)
-                        .eraseToAnyPublisher()
```

Comment:
That's true erase isn't needed.  There's a lot of places it's probably redundant.